xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h

```diff
@@ -44,8 +44,8 @@
 #define ADDON_GLOBAL_VERSION_GENERAL_XML_ID           "kodi.binary.global.general"
 #define ADDON_GLOBAL_VERSION_GENERAL_DEPENDS          "General.h"
 
-#define ADDON_GLOBAL_VERSION_GUI                      "5.12.0"
-#define ADDON_GLOBAL_VERSION_GUI_MIN                  "5.10.0"
+#define ADDON_GLOBAL_VERSION_GUI                      "5.13.0"
+#define ADDON_GLOBAL_VERSION_GUI_MIN                  "5.13.0"
```

Comment:
@MartijnKaijser is this allowed in beta phase?

Reply:
It is only my addon using this and I am ok with this bump.

Reply:
indeed VNSI is the only binary addon that uses the skin files.

Reply:
If it's okay for you then all fine.
IMO it doesn't matter which addons use it, but the question is if we want to break API or not.

Reply:
I can leave the interface functions (as dummies what they already are) in and keep the API versions. No strong feelings about this. Most likely they will stay for another 10 years or longer if we don't care now.
Let me know what solution you prefer.

Reply:
I don't block removing them, but only my addon uses it is the wrong explanation for why an API change is allowed or not.

Reply:
It depends whether we want to establish rules for the sake of having rules or take a pragmatic approach. For what reason would you block an API change if there are no real downsides?